### PR TITLE
refactor: move theme to the app_ui package

### PR DESCRIPTION
**Pull request by kamilgadek**

### CODE REVIEW

File: packages/app_ui/lib/src/theme/app_colors.dart
Comment (mszakacz) [inline packages/app_ui/lib/src/theme/app_colors.dart:1]: @kamilgadek please take a look at the example I link you in the Issue. AppColors is located in theme folder or somewhere else? The same for AppTheme and TextStyles. Please read a Issue more carefully

File: packages/app_ui/pubspec.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  google_fonts: ^6.2.1
```
Comment (mszakacz) [inline packages/app_ui/pubspec.yaml:13]: for what is that?

**Review by mszakacz — CHANGES_REQUESTED** [review]

**Comment by mszakacz** [comment]
@kamilgadek you named the PR: `refactor: move home to the app ui package` Are you sure you move home?
probably `refactor: move theme to the app_ui package` would be more correct

File: packages/app_ui/pubspec.yaml (hunk 1 above)
Comment (kamilgadek) [reply packages/app_ui/pubspec.yaml:13]: For my font style (poppins) i use it from google_fonts package

**Review by mszakacz — APPROVED** [review]
looks ok

**Pull request merged**